rpcs3/Emu/Io/usb_device.h

```diff
@@ -0,0 +1,197 @@
+#pragma once
+
+#include "Emu/Memory/vm.h"
+
+struct UsbTransfer;
+
+// Usb descriptors
+enum : u8
+{
+	USB_DESCRIPTOR_DEVICE       = 0x01,
+	USB_DESCRIPTOR_CONFIG       = 0x02,
+	USB_DESCRIPTOR_STRING       = 0x03,
+	USB_DESCRIPTOR_INTERFACE    = 0x04,
+	USB_DESCRIPTOR_ENDPOINT     = 0x05,
+	USB_DESCRIPTOR_HID          = 0x21,
+	USB_DESCRIPTOR_ACI          = 0x24,
+	USB_DESCRIPTOR_ENDPOINT_ASI = 0x25,
+};
+
+struct UsbDeviceDescriptor
+{
+	le_t<u16, 1> bcdUSB;
```

Comment:
it is kind of weird that this file uses camelCase in most instances

Reply:
The name of usb elements come from the usb spec and they're in camelCase in the spec.